spec/network-object.spec.json

```diff
@@ -29,6 +30,11 @@
       "pattern": "^0x[a-fA-F0-9]{40}$"
     },
 
+    "Transaction": {
```

Comment:
Since I've already asked for one change :)

Let's call this "class" `TransactionID` just for the sake of explicitness (and update on L9)